pandas/tests/base/test_ops.py

```diff
@@ -126,8 +126,9 @@ def check_ops_properties(self, props, filter=None, ignore_failures=False):
                     err = AttributeError
                     if issubclass(type(o), DatetimeIndexOpsMixin):
                         err = TypeError
+                        msg = "The object cannot be datatimelike"
 
-                    with pytest.raises(err):
+                    with pytest.raises(err, match=msg):
```

Comment:
I think that the CI is red because, if it's never entering the ```if``` statement above, ```msg``` never gets defined.

Reply:
Oops!Thanks @MomIsBestFriend for reviewing this.Have edited it in the latest commit.